win32/win32.h

```diff
@@ -284,8 +284,8 @@ MSVC_DIAG_RESTORE
    importing __PL_nan_u across DLL boundaries in size in the importing DLL
    will be more than the 8 bytes it will take up being in each XS DLL if
    that DLL actually uses __PL_nan_u */
-extern const __declspec(selectany) union { unsigned __int64 __q; double __d; }
-__PL_nan_u = { 0x7FF8000000000000UI64 };
+union U { unsigned __int64 __q; double __d; };
+extern const __declspec(selectany) U __PL_nan_u = { 0x7FF8000000000000UI64 };
```

Comment:
This looks like a reasonable solution based on the discussion, but I think it could use a more unique class name than "U", maybe "PerlNan".

Reply:
Yes, for sure. I wanted to change the name, but forgot about it. Thanks for reminding me 😅